_includes/scenarios/integrate/connector-events.md

```diff
@@ -2,12 +2,13 @@
 | consumption.attributeUpdated                                                         | [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#LocalAttribute)             | ... an Attribute was updated manually or through a Request.                                                                                                                                                                            |
 | consumption.incomingRequestReceived                                                  | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an incoming Request was received either by loading a Relationship Template or by receiving a Message                                                                                                                               |
 | consumption.incomingRequestStatusChanged                                             | [RequestStatusChangedEventData](#requeststatuschangedeventdata)                                | ... the status of an incoming Request has changed.                                                                                                                                                                                     |
 | consumption.messageProcessed                                                         | [MessageProcessedEventData](#messageprocessedeventdata)                                        | ... a Message was processed by Modules like the `RequestModule` or `DeciderModule`.                                                                                                                                                    |
 | consumption.outgoingRequestCreated                                                   | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an outgoing Request was created.                                                                                                                                                                                                   |
+| consumption.outgoingRequestCreatedAndCompleted                                       | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an outgoing Request was created and directly completed.                                                                                                                                                                            |
 | consumption.<br>outgoingRequestFromRelationshipCreationChange<br>CreatedAndCompleted | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an outgoing Request was created and directly completed.<br>This happens if the Response came in with a new Relationship.                                                                                                           |
```

Comment:
The event `consumption.outgoingRequestFromRelationshipCreationChangeCreatedAndCompleted` is a "convenience event" and acts as a filter for all consumption.outgoingRequestCreatedAndCompleted events.
If a consumption.outgoingRequestCreatedAndCompleted event is fired AND it was fired due to a RelationshipCreation Change, an additional `consumption.outgoingRequestFromRelationshipCreationChangeCreatedAndCompleted` is fired. This is especially useful to act on incoming RelationshipRequests.

Reply:
@Milena-Czierlinski please use this comment and make the description for outgoingRequestFromRelationshipCreationChangeCreatedAndCompleted better - I think this needs more clarification